library/src/main/java/pyxis/uzuki/live/naraeimagepicker/fragment/AllFragment.kt

```diff
@@ -50,4 +55,22 @@ class AllFragment : BaseFragment<ImageItem>() {
             }
         }
     }
+
+    private fun onImageClick(imageItem: ImageItem) {
+        if (SelectedItem.contains(imageItem)) {
+            SelectedItem.removeItem(imageItem)
+            selectCount--
+        } else {
+            selectCount++
+            addSelectedItem(imageItem)
+        }
+        sendEvent(ToolbarEvent("$selectCount Selected", true))
```

Comment:
The settings for enableUpInParentView are unlikely to be reflected.

Before: true
After: PickerSet.getSettingItem().uiSetting.enableUpInParentView

Reply:
Yes, I will do that.